src/main/kotlin/com/rooplor/classcraftbackend/utils/JsonValid.kt

```diff
@@ -0,0 +1,14 @@
+package com.rooplor.classcraftbackend.utils
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.ObjectMapper
+
+object JsonValid {
+    fun isValidJson(content: String): Boolean =
+        try {
+            ObjectMapper().readTree(content)
+            true
+        } catch (e: JsonProcessingException) {
+            false
+        }
```

Comment:
What it can do

Reply:
for validate json format when user create content